CONTRIBUTING.md

```diff
@@ -1,38 +1,75 @@
-Contributions to this repository are intended to become part of Recommendation-track documents governed by the
-[W3C Patent Policy](https://www.w3.org/Consortium/Patent-Policy-20040205/) and
-[Software and Document License](https://www.w3.org/Consortium/Legal/copyright-software). To make substantive contributions to specifications, you must either participate
+# Contributor Guidelines
+
+Thank you for your interest in contributing to the [CSS Working Group](https://www.w3.org/Style/CSS/) 
+drafts!
+
+Contributions to this repository are intended to become part of Recommendation-track 
+documents governed by the: 
+
+  * [W3C Patent Policy](https://www.w3.org/Consortium/Patent-Policy-20040205/)
+  * [Software and Document License](https://www.w3.org/Consortium/Legal/copyright-software)
+
+To make substantive contributions to specifications, you must either participate
 in the relevant W3C Working Group or make a non-member patent licensing commitment.
 
-If you are not the sole contributor to a contribution (pull request), please identify all 
-contributors in the pull request comment.
+## Issue disclosure and discussion
+
+The first step for any contribution, substantive or not, is to either:
+
+  1. [Find an existing issue](https://github.com/w3c/csswg-drafts/issues) directly related to the contribution
+  2. [Add a new issue](https://github.com/w3c/csswg-drafts/issues/new)
+
+Issues are where individual reports and Working Group discussions come together such 
+that the eventual consensus can be turned into official specification language.
 
-To add a contributor (other than yourself, that's automatic), mark them one per line as follows:
+If you are familiar with a [GitHub-based pull request contribution workflow](https://help.github.com/articles/about-pull-requests/), 
+please note that **most issues are subject to quite a bit of discussion** before 
+draft language becomes ready for a pull request. Please ensure the full problem 
+space of the issue is explored in the discussion, and that consensus has been reached 
+by the participating members, before contributing to the 
+repository.
 
-```
-+@github_username
-```
+## Normative and/or substantive contributions
 
-If you added a contributor by mistake, you can remove them in a comment with:
+The CSS Working Group operates via the consensus of its membership, and discusses 
+all significant matters prior to implementation.
 
-```
--@github_username
-```
+If adding an new, normative issue, please label/tag the PR as 'Agenda+' to bring 
+it to the Working Group's attention.
```

Comment:
Unfortunately, GitHub's access control makes labels inaccessible except to members. :( We'll have to leave this paragraph out. Or note that issues are regularly (but sometimes infrequently) triaged by the WG members responsible for the relevant area of CSS.